packages/typespec-go/test/cadlranch/payload/mediatypegroup/zz_responses.go

```diff
@@ -6,12 +6,14 @@ package mediatypegroup
 
 // MediaTypeStringBodyClientGetAsJSONResponse contains the response from method MediaTypeStringBodyClient.GetAsJSON.
 type MediaTypeStringBodyClientGetAsJSONResponse struct {
-	Value *string
+	ContentType *string
```

Comment:
I just noticed this. Why is this now showing up?

Reply:
not quite sure what this change come from? the content type header is a constant. what should go generate?

Reply:
Including the content type header for non-binary responses doesn't add any value. If tcgc is modeling this as a constant then it will be easy enough to filter out. It's just surprising that this showed up now and not months ago.

Reply:
We can't just skip it when it's a constant as that removes it from cases like binary responses where it might be useful.

Taking a step back, do we know what change is causing this to show up?

Reply:
i revisited the tcgc code. this should be by design, since use do define this head in the spec. in previous tcgc, there should be a bug that tsp compiler moved this header into the response's content type property, not put it in the parameters, the latest tcgc fix it.

Reply:
Got it, thanks for following up.